### ISSUE TITLE
Bug S1: Client phone number not being shown in booking Card

### ISSUE THREAD

**Issue by joshdanielson** [loconomics/loconomics#784]
### Bug Description
This may be more a design issue than a bug. When a client makes a free consultation booking, the service professional does not receive their email address or phone number and the address is left blank on the booking card. (we do not include the information in the booking confirmation due to HIPAA regulations). 

### Potential Fixes
- Always show the client phone number in the booking card (I think this is the easiest fix and most user friendly).
- Remove the address field if it's a phone consultation (may be hard with current architecture...could be included with offering templates).

**Comment by IagoSRL** [comment]
Highly related to #876, that applies to both phone and email for every booking.

**Comment by IagoSRL** [comment]
As part of this was duped at #876 and that fixed, I took a look to the remaining task here (hide address when irrelevant at appointment), since I had being reviewing already this piece of code and found myself with the correct mind, I tackled this and fixed:
- in a first, quick way, I fixed it for read-only display
- digging a bit, I fixed it too for the edit mode (even is disabled currently, cause of #644), in a way that is reusable for the future refactor this piece is needing.
- This last required a database modification, adding the column isRemoteService to the pricingSummaryDetail table; contains the analogous value copied from the service set-up (formerly [providerPackage].[IsPhone], I used a more proper name).
- An upgrade script set ups current values properly, and back-end code was updated to properly get and set that new value. This actually prevents some edge case problems in the future!

I have this ready, merged to *master*.
